raiden-ts/src/transfers/epics/mediate.ts

```diff
@@ -5,10 +5,25 @@ import { filter, map, withLatestFrom } from 'rxjs/operators';
 import { Capabilities } from '../../constants';
 import { RaidenAction } from '../../actions';
 import { RaidenState } from '../../state';
+import { RaidenConfig } from '../../config';
 import { RaidenEpicDeps } from '../../types';
-import { Int } from '../../utils/types';
+import { Address, Int } from '../../utils/types';
 import { transfer, transferSigned } from '../actions';
-import { Direction } from '../state';
+import { Direction, TransfersState } from '../state';
+
+function shouldMediate(
+  action: transferSigned,
+  address: Address,
+  { sent }: TransfersState,
+  { caps }: RaidenConfig,
+): boolean {
+  const is_mediation_enabled = !caps?.[Capabilities.NO_MEDIATE];
+  const is_secrethash_used = action.meta.secrethash in sent;
+  const is_target =
```

Comment:
Please, use `camelCase` for the variable names.
Also, this one would be called `isntTarget`, since it's checking `message.target !== address`, but it's also negated below.. this seems off.